tests/test_powershell.ps1

```diff
@@ -1,13 +1,25 @@
-# cd to script's directory
+# If anything fails, we want the script to stop and exit with an error code so the workflow detects the failure
+$ErrorActionPreference = 'Stop'
+
 $scriptpath = $MyInvocation.MyCommand.Path
 $dir = Split-Path $scriptpath
 cd $dir
+cd ../client/windows
 
 echo "------ Add a registry key with the client configuration ------"
-$Content = Get-Content -path "../client/windows/client.conf"
+$Content = Get-Content -path "client.conf"
 $Bytes = [System.Text.Encoding]::UTF8.GetBytes($Content)
 $Encoded = [convert]::ToBase64String($Bytes)
 reg add HKEY_LOCAL_MACHINE\SOFTWARE\Nivlheim /f /v config /t REG_SZ /d $Encoded
 
 echo "------ Run the client ------------"
-../client/windows/nivlheim_client.ps1 -certfile nivlheim.p12 -logfile nivlheim.log -server bilderbygger.uio.no -trustallcerts:1 -nosleep:1
+./nivlheim_client.ps1 -certfile nivlheim.p12 -logfile nivlheim.log -server bilderbygger.uio.no -trustallcerts:1 -nosleep:1
+if (-not $?) { exit 1 }
```

Comment:
Is it necessary to have the `if (-not $?) ...` blocks even with `$ErrorActionPreference = 'Stop'`?

LGTM, anyway.

Reply:
Yes, here's a proof:
```Powershell
# test1.ps1:
$ErrorActionPreference = "Stop"
./test2.ps1
Write-Host "Ja"

# test2.ps1:
exit 1

PS C:\Users\Admin\Documents> ./test.ps1
Hei


```

Reply:
Ooh, I see; PowerShell does not consider exit status 1 from sub-shells an error.  Thanks for the clarification, LGTM :+1: